backends/bmv2/run-bmv2-test.py

```diff
@@ -18,41 +18,65 @@
 
 from subprocess import Popen
 from threading import Thread
-import json
 import sys
-import re
 import os
-import stat
 import tempfile
 import shutil
-import difflib
-import subprocess
-import time
-import random
-import errno
-try:
-    from scapy.layers.all import *
-    from scapy.utils import *
-except ImportError:
-    pass
+import argparse
+
+from scapy.layers.all import *
+from scapy.utils import *
 from bmv2stf import RunBMV2
-import pdb
 SUCCESS = 0
 FAILURE = 1
 
 
-class Options(object):
+PARSER = argparse.ArgumentParser()
```

Comment:
I personally think that global variables are evil. Why not move this to the function using it?